app/src/main/java/com/keylesspalace/tusky/util/LinkHelper.kt

```diff
@@ -86,7 +86,7 @@ fun setClickableText(
         trailingHashtagView == null || tags.isNullOrEmpty() -> Pair(spannableContent.length, emptyList())
         else -> getTrailingHashtags(spannableContent)
     }
-    val inlineHashtags = mutableListOf<CharSequence>()
+    val inlineHashtags = mutableSetOf<CharSequence>()
```

Comment:
This is so we count hashtags only once, no matter how often they appear. Otherwise the `showHashtagBar` check below is wrong for posts that use a hashtag multiple times.